tips/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils import timezone
+from expenses.models import Expenses
+
+
+class Tip(models.Model):
+    author = models.CharField(max_length=200)
+    text = models.TextField()
```

Comment:
It may be appropriate to limit the length of the tip

Reply:
++